### ISSUE TITLE
Unable to install on mac

### ISSUE THREAD

**Issue by aburrell** [MAVENSDC/cdflib#163]
I am unable to install cdflib on my computers, apparently because I don't use homebrew.  This is how the error appears:

```
~/aburrell/git/cdflib % pip install -e .
Defaulting to user installation because normal site-packages is not writeable
Obtaining file:///~/aburrell/git/cdflib
  Installing build dependencies ... done
  Checking if build backend supports build_editable ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
  Preparing metadata (pyproject.toml) ... done
Requirement already satisfied: numpy in /opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages (from cdflib==0.4.5.dev20+g92f5d35) (1.23.0)
Requirement already satisfied: attrs>=19.2.0 in /opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages (from cdflib==0.4.5.dev20+g92f5d35) (21.4.0)
Installing collected packages: cdflib
  Running setup.py develop for cdflib
    error: subprocess-exited-with-error
    
    × python setup.py develop did not run successfully.
    │ exit code: 1
    ╰─> [11 lines of output]
        /private/var/folders/d5/d42l5vcd5pq112_vffrvs3w00000gq/T/pip-build-env-a48vfs5g/overlay/lib/python3.9/site-packages/setuptools/config/setupcfg.py:463: SetuptoolsDeprecationWarning: The license_file parameter is deprecated, use license_files instead.
          warnings.warn(msg, warning_class)
        /private/var/folders/d5/d42l5vcd5pq112_vffrvs3w00000gq/T/pip-build-env-a48vfs5g/overlay/lib/python3.9/site-packages/setuptools/config/setupcfg.py:463: SetuptoolsDeprecationWarning: The license_file parameter is deprecated, use license_files instead.
          warnings.warn(msg, warning_class)
        running develop
        /private/var/folders/d5/d42l5vcd5pq112_vffrvs3w00000gq/T/pip-build-env-a48vfs5g/overlay/lib/python3.9/site-packages/setuptools/command/easy_install.py:144: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip and other standards-based tools.
          warnings.warn(
        WARNING: The user site-packages directory is disabled.
        /private/var/folders/d5/d42l5vcd5pq112_vffrvs3w00000gq/T/pip-build-env-a48vfs5g/overlay/lib/python3.9/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
          warnings.warn(
        error: [Errno 2] No such file or directory: 'brew'
        [end of output]
    
    note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× python setup.py develop did not run successfully.
│ exit code: 1
╰─> [11 lines of output]
    /private/var/folders/d5/d42l5vcd5pq112_vffrvs3w00000gq/T/pip-build-env-a48vfs5g/overlay/lib/python3.9/site-packages/setuptools/config/setupcfg.py:463: SetuptoolsDeprecationWarning: The license_file parameter is deprecated, use license_files instead.
      warnings.warn(msg, warning_class)
    /private/var/folders/d5/d42l5vcd5pq112_vffrvs3w00000gq/T/pip-build-env-a48vfs5g/overlay/lib/python3.9/site-packages/setuptools/config/setupcfg.py:463: SetuptoolsDeprecationWarning: The license_file parameter is deprecated, use license_files instead.
      warnings.warn(msg, warning_class)
    running develop
    /private/var/folders/d5/d42l5vcd5pq112_vffrvs3w00000gq/T/pip-build-env-a48vfs5g/overlay/lib/python3.9/site-packages/setuptools/command/easy_install.py:144: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip and other standards-based tools.
      warnings.warn(
    WARNING: The user site-packages directory is disabled.
    /private/var/folders/d5/d42l5vcd5pq112_vffrvs3w00000gq/T/pip-build-env-a48vfs5g/overlay/lib/python3.9/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
      warnings.warn(
    error: [Errno 2] No such file or directory: 'brew'
    [end of output]

note: This error originates from a subprocess, and is likely not a problem with pip.
```

OS - OS X Bug Sur
Python - 3.9
Attempted install from pip, and from the GitHub repository `master` branch using pip as instructed in the README

**Comment by dstansby** [comment]
This is a really odd error message. Are you deliberately trying to install a development version? If not, `pip install cdflib` should work. I'm also wondering if you have an old checkout of the repository - could you check if your local checkout is up to date with the latest commit (92f5d355c6bfdd1105a48f4a6a597f8f1c740945)

**Comment by aburrell** [comment]
I tried that first and got the same error message.  I can verify that the GitHub `master` branch is up-to-date with the latest commit.  You get the same error whether or not you install with or without the develop option.  I was hoping to be able to poke around and see what was causing the error, but I don't see anything obviously set to depend on brew.

**Comment by dstansby** [comment]
Weird, I have no idea why this is happening. There's no mention of `brew` anywhere in the source code, and I've checked and all the setup configuration files seem fine. Can you `pip install` other packages fine?

**Comment by aburrell** [comment]
Yup, just did another one this morning.  No issue.  And I've had no issue with this in the past for this package.

**Comment by bryan-harter** [comment]
Can you pip install older versions of cdflib?  For example, pip install cdflib==0.3.20?  If that works, then try 0.4.0, then 0.4.1, etc.  I think we're on 0.4.3.  That would help us track down what might've changed.

**Comment by dstansby** [comment]
If you run `python -m pip install -e .` does it work? I wonder if there's some unintended cross talk between different Python installations, and `python -m pip` will ensure your pip installation matches the python installation.

**Comment by JonathonMSmith** [comment]
I have no idea what's going on here. I uninstalled brew and was able to install this by cloning and running `pip install -e .`
Jeff and I saw some weird things happening. 

Your error output shows `Defaulting to user installation because normal site-packages is not writeable` which I don't see.

Next setuptools_scm is generating the wrong version number `0.4.5`, this also happens on my machine.

lastly, all of these issues seem to be coming at first from setuptools deprecations.

I have setuptools v 61.2.0 on my brewless machine that successfully installed cdflib.

Maybe something here is important I don't really know
